docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -90,13 +71,11 @@ EOF
 aws iam create-policy --policy-name KroxyliciousRecordEncryption --policy-document file:///tmp/policy
 ----
 
-== Apply the alias-based policy to the filter's application identity
+== Establish an authentication mechanism for the filter
+
+The filter must authenticate to AWS in order to perform envelope encryption operations, such as generating and decrypting DEKs.
+
+include::con-aws-kms-setup-application-identity-iam.adoc[leveloffset=+1]
+include::con-aws-kms-setup-application-identity-ec2-metadata.adoc[leveloffset=+1]
```

Comment:
We could conditionalize to leave out of downstream here too

Reply:
done